src/gui/mainwindow.cpp

```diff
@@ -262,7 +309,9 @@ MainWindow::MainWindow(QWidget *parent)
     connect(m_ui->actionIncreasePriority, SIGNAL(triggered()), m_transferListWidget, SLOT(increasePrioSelectedTorrents()));
     connect(m_ui->actionDecreasePriority, SIGNAL(triggered()), m_transferListWidget, SLOT(decreasePrioSelectedTorrents()));
     connect(m_ui->actionBottomPriority, SIGNAL(triggered()), m_transferListWidget, SLOT(bottomPrioSelectedTorrents()));
+#ifndef Q_OS_MAC
     connect(m_ui->actionToggleVisibility, SIGNAL(triggered()), this, SLOT(toggleVisibility()));
```

Comment:
@vit9696:  Not directly relevant to this PR, but do you think you could add a commit updating the `connect` syntax used in `mainwindow.cpp` to the Qt5 standard?  

There should be a dedicated function for that in Qt Creator, if I recall correctly.